### DISCUSSION TITLE
Restart HA without config check

### DISCUSSION THREAD

**Discussion by PiotrMachowski** [frenck/spook#6] [Ideas] [answered]
Right now `homeassistant.restart` service performs config check before restarting. Adding a way to skip checking might fit *Spook doesn't care* approach.

Thank you for this integration!

**Comment by frenck** [comment, marked as answer]
![CleanShot 2023-06-03 at 14 12 46](https://github.com/frenck/spook/assets/195327/eadc8863-4738-4e05-b778-e47757126ca2)

**Reply by frenck** [reply]
PR: https://github.com/frenck/spook/pull/213

**Reply by frenck** [reply]
It extends the existing `homeassistant.restart` service, by adding a `force` option, which defaults to `false`.

To force it:

```yaml
- service: homeassistant.restart
  data:
    force: true
```

It will skip config checks and even ignores database migrations in progress. It will kill HA asap and restart.

⚠️ this is a bad idea of course :)

**Reply by PiotrMachowski** [reply]
Perfect 👍
